src/style.css

```diff
@@ -0,0 +1,146 @@
+/* #region fonts */
+@font-face {
+  font-family: "Avenir";
+  font-weight: lighter, normal, bold;
+  src: url("./fonts/Avenir-Book.ttf"),
+    url("./fonts/Avenir-Heavy.ttf");
+}
+
+/* #endregion */
+
+/* #region settings */
+html,
+body,
+section,
+div,
+input,
+label,
+form {
+  padding: 0;
+  margin: 0;
+}
+
+.search-container {
+  margin: 8px;
+}
+/* #endregion */
+
+/* #region variables */
+:root {
+  --avenir: "Avenir";
+  --main-color: rgba(61, 78, 97, 1);
+  --focus-first: rgb(225, 231, 237);
+  --focus-second: rgb(255, 255, 255);
+  --big-input-height: 22px;
+  --small-input-height: 19px;
+}
+/* #endregion */
+
+/* #region form */
+.search-form {
+  box-sizing: border-box;
+  position: relative;
+  border: 1px solid rgba(225, 231, 237, 1);
+  border-radius: 4px;
+  margin: 20px auto 0;
+  font-family: var(--avenir);
+  background-color: rgba(255, 255, 255, 1);
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+
+  transition: 0.3s ease;
+  -moz-transition: 0.3s ease;
+  -webkit-transition: 0.3s ease;
+}
+
+.search-form:focus-within {
+  background: linear-gradient(
+    to bottom,
+    rgba(255, 255, 255, 1) 0%,
+    rgba(246, 246, 247, 1) 100%
+  );
+}
+
+.search-form:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.search-form-big {
+  width: 100%;
+  height: 70px;
+  padding-inline: 62px;
+}
+
+.search-form-small {
```

Comment:
instead of creating this create one base class with all styles needed for big input, some styles are similar in big and small inputs, so you can just create .search-form--small and add it with a base class to the small input, so it will override needed styles